provision.sh

```diff
@@ -27,6 +27,9 @@ sh -c 'echo SUBSYSTEM==\"usb\", ATTRS{idVendor}==\"1915\", ATTRS{idProduct}==\"7
 sh -c 'echo SUBSYSTEM==\"usb\", ATTRS{idVendor}==\"1915\", ATTRS{idProduct}==\"0101\", MODE=\"0664\", GROUP=\"plugdev\" >> /etc/udev/rules.d/99-crazyradio.rules'
 sh -c 'echo SUBSYSTEM==\"usb\", ATTRS{idVendor}==\"0483\", ATTRS{idProduct}==\"5740\", MODE=\"0664\", GROUP=\"plugdev\" >> /etc/udev/rules.d/99-crazyflie.rules'
 
+# Add user to dialout group, to give access to serial ports
+usermod -a -G dialout $USER
```

Comment:
I think this script is executed as root. `$USER` needs to be replaced with the user used in the vm: `bitcraze`

Reply:
Are you sure? The provision-json has this magic:

```
{
      "execute_command": "echo '{{user `password`}}' | {{ .Vars }} sudo -E -S sh '{{ .Path }}'",
      "script": "provision.sh",
      "type": "shell"
    }
```

And earlier in this script we have:

```
# Adding udev rules for Crazyradio and Crazyflie
usermod -a -G plugdev $USER
[...]
```

Reply:
Hum .... no I am not so sure anymore. Plugdev definitely work so you should be right.